src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -207,6 +208,18 @@ public File clickExportFields() throws Exception
         return exportFiles[0];
     }
 
+    public DomainFormPanel clickDeleteFields()
+    {
+        getWrapper().scrollIntoView(elementCache().deleteFieldsButton);
+        elementCache().deleteFieldsButton.click();
+
+        ModalDialog confirmDialog = new ModalDialog.ModalDialogFinder(getDriver())
+                .withTitle("Confirm Delete Selected Fields").timeout(1000).waitFor();
+        confirmDialog.dismiss("Yes, Delete Fields");
```

Comment:
Would there ever be a test case where you want to just return the dialog and not explicitly click "yes"?

Reply:
In future might be